generator/internal/codegen_utils.cc

```diff
@@ -130,12 +133,27 @@ ProcessCommandLineArgs(std::string const& parameters) {
         "--cpp_codegen_opt=googleapis_commit_hash=<hash> must be specified.");
   }
 
+  auto copyright_year =
+      std::find_if(command_line_args.begin(), command_line_args.end(),
+                   [](std::pair<std::string, std::string> const& p) {
+                     return p.first == "copyright_year";
+                   });
+  auto current_year =
+      absl::FormatTime("%Y", absl::Now(), absl::LocalTimeZone());
```

Comment:
I would avoid calling `absl::Now()` in a function, preferring to inject a value from the caller (probably back to `main()`).